Lib/test/test_pathlib.py

```diff
@@ -11,7 +11,7 @@
 
 from test import support
 android_not_root = support.android_not_root
-TESTFN = support.TESTFN
+from test.support import (can_symlink, skip_unless_symlink, TESTFN)
```

Comment:
`test.support` is already imported as `support`, so please just access the attributes off of that instead of importing them directly.

And FYI, the parentheses are unnecessary.